compiler/optimizer/PreExistence.hpp

```diff
@@ -27,6 +27,10 @@
 #include "env/KnownObjectTable.hpp"
 #include "env/TRMemory.hpp"
 
+// Temporary macro to coordinate changes between omr and openj9.
+// This will eventually be changed to TR_LogTracer.
+#define TR_PREXARGINFO_TRACER_CLASS TR_InlinerTracer
```

Comment:
Is there an issue tracking this work?

Reply:
There is only this issue in in openj9: https://github.com/eclipse/openj9/issues/7936. Should I create an issue in omr to track this? Otherwise, I can update the openj9 issue with a checklist of what steps are done/need to be done.

Reply:
No, that's fine. Just wanted to have a written reference of where the work is being tracked so we don't forget about it. Thanks!